src/api/api.types.tsx

```diff
@@ -101,3 +101,64 @@ export interface System extends CreatedModifiedMixin {
 }
 
 export interface SystemPatch extends Partial<SystemPost> {}
+
+// ------------------------------------ CATALOGUE CATEGORIES ------------------------------------
+
+export interface AllowedValuesList {
+  type: 'list';
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  values: any[];
+}
+export type AllowedValues = AllowedValuesList;
+
+export enum CatalogueCategoryPropertyTypes {
```

Comment:
```suggestion
export enum CatalogueCategoryPropertyType {
```
Types are usually singular and this matches the backend.